src/udp/mod.rs

```diff
@@ -1,4 +1,4 @@
-pub use self::hole_punch::UdpHolePunchMediator;
+pub use self::hole_punch::{UdpHolePunchMediator, UdpRendezvousClient};
```

Comment:
I'm not comfortable exposing stuff only from a crate only for the purposes of testing. Maybe the testing code should be designed in some other way but the crate should not become a "debug-library" where all the internals are exposed to the user and not for any non-trivial reason.